anvil_consortium_manager/templates/anvil_consortium_manager/index.html

```diff
@@ -104,39 +104,6 @@ <h5 class="card-header"><span class="fa-solid fa-user-group mx-2"></span>Managed
         </div>
       </div>
 
-
-      {% for workspace_type, workspace_name in registered_workspaces.items %}
-      <div class="col">
-        <div class="card shadow-sm">
-          <h5 class="card-header"><span class="fa-solid fa-computer mx-2"></span>{{ workspace_name }}s</h5>
-          <div class="card-body">
-            <p>
-              Interact with {{ workspace_name }}s on AnVIL.
-              {% if perms.anvil_consortium_manager.anvil_project_manager_edit %}You can create or import workspaces and then share them with a group.{% endif%}
-            </p>
-          </div>
-
-          <ul class="list-group list-group-flush">
-            <li class ="list-group-item">
-              <a href="{% url 'anvil_consortium_manager:workspaces:list' workspace_type %}" class="icon-link">List workspaces</a>
-              <span class="fa-solid fa-angle-right"></span>
-            </li>
-            {% if perms.anvil_consortium_manager.anvil_project_manager_edit %}
-            <li class ="list-group-item">
-              <a href="{% url 'anvil_consortium_manager:workspaces:new' workspace_type %}" class="icon-link">Add a workspace</a>
-              <span class="fa-solid fa-angle-right"></span>
-            </li>
-            <li class ="list-group-item">
-              <a href="{% url 'anvil_consortium_manager:workspaces:import' workspace_type  %}" class="icon-link">Import a workspace from AnVIL</a>
-              <span class="fa-solid fa-angle-right"></span>
-            </li>
-            {% endif %}
-          </ul>
-
-        </div>
-      </div>
-      {% endfor %}
-
```

Comment:
I think we want a general "Workspaces" card that has the following links:
- Link to the workspace landing page
- Link to the list of all workspaces
- Link to the WorkspaceGroupSharing list

so similar to the links in the navbar.